craco.config.cjs

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-env node */
```

Comment:
This has been broken out into its own PR: https://github.com/Uniswap/interface/pull/6311